src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -880,9 +880,15 @@ private void UpdateTopResult(SearchMemberGroup memberGroup)
             }
 
             var topMemberGroup = new SearchMemberGroup(memberGroup.FullyQualifiedName);
-            topMemberGroup.AddMember(memberGroup.Members.First());
 
+            // Clone top node.
+            var topNode = new NodeSearchElementViewModel(memberGroup.Members.FirstOrDefault());
```

Comment:
After talking to @aosyatnik, I think the design intent is to have *Top Result* separately highlightable (i.e. it should be selectable on its own). So cloning makes sense for now (though I think it over complicates the code).